docs/installation/mac.md

```diff
@@ -50,134 +44,192 @@ Docker container using standard localhost addressing such as `localhost:8000` or
 ![Linux Architecture Diagram](/installation/images/linux_docker_host.svg)
 
 In an OS X installation, the `docker` daemon is running inside a Linux virtual
-machine provided by Boot2Docker.
+machine.
 
 ![OSX Architecture Diagram](/installation/images/mac_docker_host.svg)
 
-In OS X, the Docker host address is the address of the Linux VM.
-When you start the `boot2docker` process, the VM is assigned an IP address. Under
-`boot2docker` ports on a container map to ports on the VM. To see this in
+In OS X, the Docker host address is the address of the Linux VM. When you start
+the VM with `docker-machine` it is assigned an IP address. When you start a
+container, the ports on a container map to ports on the VM. To see this in
 practice, work through the exercises on this page.
 
 
 ### Installation
 
-1. Go to the [boot2docker/osx-installer ](
-   https://github.com/boot2docker/osx-installer/releases/latest) release page.
-
-4. Download Boot2Docker by clicking `Boot2Docker-x.x.x.pkg` in the "Downloads"
-   section.
-
-3. Install Boot2Docker by double-clicking the package.
-
-    The installer places Boot2Docker and VirtualBox in your "Applications" folder.
-
-The installation places the `docker` and `boot2docker` binaries in your
-`/usr/local/bin` directory.
-
+If you have Virtual Box running, you must shut it down before running the
+installer. 
 
-## Start the Boot2Docker Application
+1. Go to the [Docker Toolbox](https://docker-toolbox.azurewebsites.net/) release page.
 
-To run a Docker container, you first start the `boot2docker` VM and then issue
-`docker` commands to create, load, and manage containers. You can launch
-`boot2docker` from your Applications folder or from the command line.
+4. Click "DOWNLOAD DOCKER TOOLBOX."
 
-> **NOTE**: Boot2Docker is designed as a development tool. You should not use
->  it in production environments.
+3. Install Docker Toolbox by double-clicking the package or by right-clicking
+and choosing "Open" from the pop-up menu.
 
-### From the Applications folder
+    The installer launches the "Install Docker Toolbox" dialog.
+    
+    ![Install Docker Toolbox](/installation/images/mac-welcome-page.png)
```

Comment:
I actually got a confirmation page before this (this installer is using a program to determin if this software can be installed on your computer); not sure if we should go in such detail, because it's a standard dialog